locations/spiders/pollotropical.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import re
+from locations.items import GeojsonPointItem
+
+class QuickleesSpider(scrapy.Spider):
+    name = "pollotropical"
+    allowed_domains = ["https://www.pollotropical.com/"]
```

Comment:
You're only asking scrapy to go to one URL which you ave specfified. Probably overkill to set allowed_domains!